k8s/postgresql/README.md

```diff
@@ -115,10 +115,12 @@ export APP_INSTANCE_NAME=postgresql-1
 export NAMESPACE=default
 ```
 
-Configure the container image:
+Configure the container images:
 
 ```shell
+TAG=9.6
 export IMAGE_POSTGRESQL=marketplace.gcr.io/google/postgresql9:9.6-kubernetes
```

Comment:
Shouldn't this be `export IMAGE_POSTGRESQL=marketplace.gcr.io/google/postgresql9:${TAG}-kubernetes`?

Reply:
Actually, this is republished image, so this should be `export IMAGE_POSTGRESQL=marketplace.gcr.io/google/postgresql:${TAG}`.